src/cascadia/TerminalApp/AppActionHandlers.cpp

```diff
@@ -312,4 +312,20 @@ namespace winrt::TerminalApp::implementation
         }
         args.Handled(true);
     }
+
+    void TerminalPage::_HandleToggleTabSwitcher(const IInspectable& /*sender*/,
+                                                const TerminalApp::ActionEventArgs& args)
+    {
+        if (const auto& realArgs = args.ActionArgs().try_as<TerminalApp::ToggleTabSwitcherArgs>())
+        {
+            // TODO: Maybe instead of passing in the TerminalApp::AnchorKey, we can just pass
+            // in a VirtualKey
+            auto anchorKey = realArgs.Key();
+            CommandPalette().ToggleTabSwitcher(anchorKey);
+            CommandPalette().Visibility(CommandPalette().Visibility() == Visibility::Visible ?
+                                            Visibility::Collapsed :
+                                            Visibility::Visible);

```

Comment:
I think we discussed this offline, but it might be less confusing if the control were named something more generic to accomodate both the CommandPalette and the TabSwitcher. I'm reusing the CommandPalette control, but just replacing the actions it pulls from to the TabSwitcher's actions. So, that's why the function calls are still coming off the CommandPalette control here.